packages/web3torrent/package.json

```diff
@@ -126,6 +126,7 @@
     "@types/webtorrent": "0.107.0",
     "@types/workbox-webpack-plugin": "5.1.0",
     "bittorrent-tracker": "9.15.0",
+    "canvas": "2.6.1",
```

Comment:
Is this actually used?

Reply:
We call `useContext` in our unit tests (via component instantiation). Without that dev dependency, the unit tests pass but we see a fairly verbose warning.